tools/cldr/cldr-to-icu/README.txt

```diff
@@ -0,0 +1,71 @@
+Basic instructions for running the LdmlConverter via Maven
+----------------------------------------------------------
+
+Note that these instructions do not currently support configuration of the converter for things
+such as limiting the set of files produced. That is supported in code and could be easily added
+to the binary, or encapsulated via an Ant task, but currently it is not directly supported.
+See the IcuConverterConfig class for the API by which this can be supported.
+
+
+Notes on JDK versions:
+----------------------
+
+It is possible that some JDK versions will not, by default, support the set of features necessary
+in the SAX parser (XML parser). If you see the message:
+
+[ERROR] ------------------------------------------------------------------
+[ERROR] Unknown error reading /<path-to-xml-file>
+[ERROR]
+[ERROR] This can sometimes be caused by using a version of Java which does
+[ERROR] not support all the XML parsing features required by this tool.
+[ERROR] Try setting the JAVA_HOME environment variable to a different Java
+[ERROR] release, if possible.
+[ERROR] ------------------------------------------------------------------
+
+Then you should see if it's possible to use a different Java runtime environment by setting
+JAVA_HOME environment variable. In Unix, this is possible by prefixing the command with a
+local assignment, such as:
+
+JAVA_HOME='<path-to-alternate-jvm>' mvn exec:java ...
+
+To run the converter tool, you need to know the locations of several key directories before
+hand. These are:
+
+
+Important directories:
+----------------------
+
+<CLDR_DIR>   = The root directory of the CLDR release.
+<ICU_DIR>    = The root directory of the ICU release (probably a parent directory of where
+               this README is located).
+<DTD_CACHE>  = The temporary cache directory in which DTD files are downloaded (this is the
```

Comment:
Caching should not be needed at all, see [CLDR-13165] (I am deleting the caching entity resolver from CLDR.)

Reply:
"Should not" eventually ... or "should not" now ?

Reply:
- Already has no effect as of current CLDR / ICU master, as no DTDs are loaded via http/s
> `find ~/src/cldr ~/src/icu -name '*.xml' | fgrep .dtd | fgrep http   # no results`
- CachingEntityResolver is being removed in https://github.com/unicode-org/cldr/pull/108 (including from the API)

Reply:
Yay, I'll see about getting all the code and docs related to this stripped out of the API code before I push another version of this PR. Thanks!

Reply:
I may have misunderstood you. I thought you were saying that DTD_CACHE is no longer needed. It doesn't work for me when I omit it.

I suspect it is still needed when dealing with older releases at the moment (and since I am heavily testing on CLDR-35.1 / ICU-64.2 since that's the only release data I can test on) I need to leave it there for now. We can remove it again once the dust has settled and people are happy that we will never need to run the new tools on an old release of the data.

Reply:
fair enough, but I expect to merge the PR into CLDR 36.